### DISCUSSION TITLE
use rosetta simulator in the lane - fastlane snapshot

### DISCUSSION THREAD

**Discussion by yellowTiger** [fastlane/fastlane#22140] [Q&A]
as above，my MacBook is use m1 chip，I need rosseta run my project successfully. Terminal will not select  rosetta to run the simulator，can anyone help me？

**Comment by jennifer986block** [comment]
Hello,
If you're having trouble running your project on Rosetta with your M1 MacBook, you can force the Terminal to use Rosetta. Here are the steps: [MyWakeID Net](https://www.mywakeid.com)

Find the Terminal app: Go to Applications > Utilities, and locate the Terminal app.

Get Info: Right-click on the Terminal app and select "Get Info". 3. Enable Rosetta: Check the box labeled "Open using Rosetta".

This will ensure your Terminal uses Rosetta to run your project. If you continue to have issues, you can also use the following command in Terminal to install Rosetta if it isn't already installed:

sh
softwareupdate --install-rosetta --agree-to-license
Hope this helps, and let me know if you need further assistance! 

Best regards, 
Jennifer Block

**Reply by yellowTiger** [reply]
Thank you! You are really kind and helpful.
